diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/WhiteSpaceRule.kt

```diff
@@ -200,4 +255,12 @@ class WhiteSpaceRule : Rule("horizontal-whitespace") {
 
     private fun ASTNode.removeIfWhiteSpace() = takeIf { it.elementType == WHITE_SPACE && !it.textContains('\n') }
             ?.let { it.treeParent.removeChild(it) }
+
+    private fun getDescription(shouldBefore: Boolean,
+                               shouldAfter: Boolean,
+                               before: Int?,
+                               after: Int?): String =
+            (if (shouldBefore) " $before space(s) before" else "") +
```

Comment:
please, I am begging you - make it more readable and less compact here:
if (should && should) {
} else if (should) {
} else if (should) {

Reply:
OK, will fix.